src/services/jwtService.js

```diff
@@ -0,0 +1,42 @@
+const jwt = require('jsonwebtoken');
+
+require('dotenv').config();
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '900s',
```

Comment:
Consider increasing the `expiresIn` duration for the access token to a more typical value, such as '15m' or '1h', depending on your application's requirements. The current '900s' (15 minutes) is reasonable, but ensure it aligns with your security policies.